sections/queueing_model/subsections/performance_measures/blocking_time/main.tex

```diff
@@ -0,0 +1,225 @@
+\subsubsection{Blocking time}
+
+% TODO: Possibly replace the contents of this section with the ones for the 
+% closed form formula of the blocking time
+% Currently the direct approach one is shown
+
+Unlike the waiting time, the blocking time is only calculated for individuals of
+the second type.  
+That is because individuals of the first type cannot be blocked. 
+Thus, one only needs to consider the pathway of type 2 individuals to get the 
+mean blocking time of the system. 
```

Comment:
After this paragraph we need to write the "formula" and by that I mean: the linear system `b(u,v)=...`. (Equation `eq:general_blocking_time_at_each_state`) and that it can be solved by considering the linear system `Zx=y` where `Z_{ij} =...`, `x_{i}=...` and `y_j = ...`.

Then we move/replicate everything else in an appendix and include a similar numerical example.